validate-play-store-description.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Checks if play store descriptions does not exceed char limits
+# https://support.google.com/googleplay/android-developer/answer/113469?hl=en
+
+cd fastlane/metadata/android
+
+error=0
+
+for folder in *
+do
+    cd "$folder"
+    if [ -f "full_description.txt" ]
+    then
+        chars=$(wc -m full_description.txt | cut -d " " -f 1)
+        if [ "$chars" -gt 4000 ]
+        then
+            echo "Full descriptions in $folder exceeds 4000 char limit"
+            let error++
+        fi
+    fi
+    if [ -f "short_description.txt" ]
+    then
+        chars=$(wc -m short_description.txt | cut -d " " -f 1)
+        if [ "$chars" -gt 80 ]
+        then
+            echo "Short descriptions in $folder exceeds 80 char limit"
+            let error++
+        fi
+    fi
```

Comment:
Should you check if there is only either or none of those files?

Reply:
I have also thought about this and I think no: You can see in crowdin which parts are translated and which not.

Reply:
My question is: How will a listing look like if there is only the full description? for example...